package.json

```
@@ -1,6 +1,6 @@
 {
-  "name": "keplr",
-  "description": "Keplr is a browser extension wallet for the Inter blockchain ecosystem.",
+  "name": "Fetch Wallet",
```

Spaces aren't allowed in the npm package name. I would suggest making this consistent with the new repo name:
```suggestion
  "name": "fetch-wallet",
```

This name won't be printed anywhere that users will see.